src/auto-evo/simulation/food_source/CompoundFoodSource.cs

```diff
@@ -26,7 +26,7 @@ public override float FitnessScore(Species species, SimulationCache simulationCa
 
         var compoundUseScore = EnergyGenerationScore(microbeSpecies, compound);
 
-        var energyCost = simulationCache.GetEnergyBalanceForSpecies(microbeSpecies, patch).FinalBalanceStationary;
+        var energyCost = simulationCache.GetEnergyBalanceForSpecies(microbeSpecies, patch).TotalConsumptionStationary;
 
         return compoundUseScore / energyCost;
     }
```

Comment:
That hasn't changed much, but it makes me realize that the fitness score seems not to be actually dependent on sessility nor speed. However, this is arguably very important when scavenging for compound food.

I could get that you'd deem this outside of the scope of the PR, but perhaps it could be worth adding a small comment on this by the way?

Reply:
What if I make an issue instead? TODO comments are infamous for never being addressed, and if we make a comment for every scientific inaccuracy in auto evo you're not going to be able to see the code :)

Reply:
I made a new issue to cover this https://github.com/Revolutionary-Games/Thrive/issues/2918

Reply:
You are correct, although that's not only about scientific accuracy, but also matching experience from gameplay and auto-evo. I'm happy with this though.